035-refactor-client-examples

```diff
@@ -0,0 +1,65 @@
+# Refactor KafkaConfig files in Strimzi Client Examples
+
+This proposal is about simplifying the KafkaConfig files of the Java modules of the [Strimzi Client Examples repo](https://github.com/strimzi/client-examples). 
+
+## Current situation
+
+The Kafka*Config files of the Java modules in the [Strimzi Client Examples repo](https://github.com/strimzi/client-examples) have become quite complex since they support the configuration of several specific Kafka client properties via environment variables. 
```

Comment:
```suggestion
The `Kafka*Config` files of the Java modules in the [Strimzi Client Examples repo](https://github.com/strimzi/client-examples) have become quite complex since they support the configuration of several specific Kafka client properties via environment variables. 
```